test/api/application/fixtures.go

```diff
@@ -0,0 +1,67 @@
+package application
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/api/client"
+)
+
+var (
+	// Setup Hub API client
+	Client = client.Client
+)
+
+//
+// Set of valid Application resources for tests and reuse.
+// Invalid application for negative tests are expected to be defined within the test methods, not here.
+func Samples() (samples []api.Application) {
+	samples = []api.Application{
+		{
+			Name:        "Pathfinder",
+			Description: "Tackle Pathfinder application.",
+			Repository: &api.Repository{
+				Kind:   "git",
+				URL:    "https://github.com/konveyor/tackle-pathfinder.git",
+				Branch: "1.2.0",
+			},
+		},
+		{
+			Name: "Minimal application",
+		},
+	}
+	return
+}
```

Comment:
Maybe, instead of having `Samples()` like this. We create a public function that returns a single application. For example `MinimalApp()` or `PathfinderGit()` that can be reused.

It might not be meaningful now, but we could consider a helper function that accepts a slice of applications and just creates them. I would expect this would be helpful at the start of scenarios.

Reply:
I prefer this as oppose to calling something like `Samples[0]` in places where you wish to use the Pathfinder sample.

Reply:
Used Map for fixtures for now. Will check if other options fit better in followup tests development.